src/app/components/header/index.stories.jsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react'; // eslint-disable-line import/no-extraneous-dependencies
+import BBCNewsBanner from './index';
```

Comment:
You're referring to `BBCNewsBanner` in this file. You'll want to update these to `Header`

Reply:
Done